wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1805,3 +1807,32 @@ class ImplicitPrimitiveViolation(ASTViolation):
 
     code = 446
     error_template = 'Found implicit primitive in a form of `lambda`'
+
+
+@final
+class ApproximateConstantViolation(ASTViolation):
+    """
+    Forbids to use approximate constants.
+
+    Reasoning:
+        This constants are already defined.
+        No need to write them again, use existing values.
+
+    Solution:
+        Use constants from math module.
+        https://docs.python.org/3/library/math.html#constants
+
+    Example::
+
+        # Correct:
+        from math import pi
+
+        # Wrong:
+        pi = 3.14
+
+    .. versionadded:: 0.13.0
+
+    """
+
+    code = 447
+    error_template = 'Found approximate constant'
```

Comment:
`'Found approximate constant: {0}'` and then insert the value that is found via formatting